src/data/tests/ContentHighlightsTestData/testCourseAggregation.json

```diff
@@ -0,0 +1,6 @@
+{
+    "course:HarvardX+CS50W": true,
+    "course:HarvardX+CS50AI": true,
+    "course:HarvardX+CS50P": true,
+    "course:HarvardX+CS50x": true
+}
```

Comment:
nit: The intent of this file isn't immediately obvious. Is there a more explicit file name beyond "course aggregation" to indicate what this is used for?